test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -42,8 +42,22 @@ class test_ucp_sockaddr : public ucp_test {
                                                          multi-threading */
         CONN_REQ_TAG,                                 /* Accepting by ucp_conn_request_h,
                                                          send/recv by TAG API */
-        CONN_REQ_STREAM                               /* Accepting by ucp_conn_request_h,
+        CONN_REQ_STREAM,                              /* Accepting by ucp_conn_request_h,
                                                          send/recv by STREAM API */
+        /*
+         * double parameters for CM wireup/close protocol.
+         * TODO: remove all below then it's enabled by default
+         */
```

Comment:
can we use MT and CM as flags, in conjunction with CONN_REQ_TAG and CONN_REQ_STREAM?

Reply:
technically - yes but anyway it will be another test parameter N which will be harder to translate from test name "A/test.case/N" to enum value or their conjunction

Reply:
A/test.case/N is not the enum value anyway, it's the index of the instantiation.
let's use flags

Reply:
ok